### DISCUSSION TITLE
topic_model.transform(training_data) results in KeyError: 6467

### DISCUSSION THREAD

**Discussion by registjl** [MaartenGr/BERTopic#2160] [Q&A]
Hi -- I'm new to NLP and loving BERTopic!

I've created a model 'topic_model'.  topic_model.fit_transform(training_data) works just fine and results in a model that I've saved to disk.

When I load the model in another script, I ran the 'training_data' back through the model to see what I get the same results.
-  'fit_transform' runs ok.  
- However, topic_model.transform(training_data) returns KeyError: 6467.

Any ideas are greatly appreciated!
JLR

**Comment by MaartenGr** [comment]
Could you perhaps open up an issue instead of a discussion and fill in the necessary information? That will help me understand the problem better.

**Reply by registjl** [reply]
Will do.  I'm new to github, too ... your patience is appreciated!
